ui/reactAngularAdaptor.js

```diff
@@ -55,3 +66,24 @@ function reactAddAppointmentController($rootScope, $location, $scope, $state, ng
     $scope.appointmentParams = $stateParams.appointment;
     $scope.urlParams = $location.$$search;
 }
+
+function reactAppointmentSummaryController($rootScope, $location, $scope, $state, appService) {
+
+    $scope.setViewDate = function (date) {
+        $state.params.viewDate = date;
+    };
+    $scope.goToListView = function (date, uuids) {
+        var params = {
+            viewDate: moment(date).toDate(),
+            filterParams: {statusList: _.without(Bahmni.Appointments.Constants.appointmentStatusList, "Cancelled")}
+        };
+        if (uuids && uuids.length !== 0) {
+            params.filterParams.serviceUuids = uuids;
+        }
+        $state.go('home.manage.appointments.list', params);
+    }
+    $scope.state = $state;
+    $scope.currentProvider = $rootScope.currentProvider;
+    $scope.fullSummary = appService.getAppDescriptor().getConfigValue('enableDetailedSummaryView')
```

Comment:
@Phanindra-tw what if enableDetailedSummaryView set to false ? are we rendering the old UI ? or new reactUI with just the service summary view ?